Text/TextAnalyticsForHealth/TextAnalyticsForHealth.cs

```diff
@@ -0,0 +1,189 @@
+using System;
```

Comment:
Add copyright note to the beginning of this file like we have in other power skills:

// Copyright (c) Microsoft. All rights reserved.  
// Licensed under the MIT License. See LICENSE file in the project root for full license information.